polling_stations/templates/base.html

```diff
@@ -23,6 +23,7 @@
 
         {% block site_meta %}
             <script src="https://code.jquery.com/jquery-3.6.0.min.js"></script>
+            <script src="https://cdn.jsdelivr.net/npm/promise-polyfill@8/dist/polyfill.min.js"></script>
```

Comment:
You don't need to use the CDN if you're installing it with npm.

Can you add a new javascript file to pipeline, copying the dashboard here:

https://github.com/DemocracyClub/UK-Polling-Stations/blob/master/polling_stations/settings/static_files.py#L45-L50

And then implement it like:

https://github.com/DemocracyClub/UK-Polling-Stations/blob/faf0ad38d24f78b2d46e45a561fcbe08682db19a/polling_stations/apps/dashboard/templates/dashboard/postcode.html#L14

But only on the pages where we're actually going to use it. I'm keen on not forcing all our users to download this script for every page load, when it's mostly not going to be used.

Reply:
Actually, same for jQuery. We're already installing jQuery locally so we shouldn't be using the CDN version. I know that's not added in this PR, but it would be nice to get it out of every page load if we can!